test/e2e/performanceprofile/functests/0_config/test_suite_performance_config_test.go

```diff
@@ -59,6 +64,13 @@ func TestPerformanceConfig(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	Expect(testclient.ClientsEnabled).To(BeTrue())
+	// create test namespace
+	if err := testclient.Client.Create(context.TODO(), namespaces.TestingDaemonsetNamespace); !errors.IsAlreadyExists(err) {
+		Expect(err).ToNot(HaveOccurred())
+	}
+
+	err := daemonset.CreateForTesting(testclient.DataPlaneClient, testutils.TestingDaemonsetNamespace, testutils.TestingDaemonsetName, images.Test())
```

Comment:
let's try be more explicit about what `daemonset.CreateForTesting` is doing.
what about `CreateNodeInspector`? and adding a short comment that explains what it's doing